org.eclipse.winery.generators.ia/src/main/java/org/eclipse/winery/generators/ia/Generator.java

```diff
@@ -236,14 +238,7 @@ private void generateJavaFile(File javaService) throws IOException {
                     // Generate @WebParam
                     sb.append("@WebParam(name=\"" + parameterName + "\", targetNamespace=\"" + this.namespace + "\") ");
 
-                    // Handle required and optional parameters using @XmlElement
```

Comment:
Don't you need this logic in some way?

Reply:
No, this does not work. In every generated IA, one has to remove the `@XmlElement(required=true)` from the parameters after generation. Otherwise, the IA cannot be executed by OpenTOSCA.